recipes/LibriSpeech/ASR/transformer/hparams/conformer.yaml

```diff
@@ -29,23 +32,27 @@ data_folder_rirs: !ref <data_folder>
 train_splits: ["train-clean-100", "train-clean-360", "train-other-500"]
 dev_splits: ["dev-clean"]
 test_splits: ["test-clean", "test-other"]
+skip_prep: False
 train_csv: !ref <data_folder>/train.csv
 valid_csv: !ref <data_folder>/dev-clean.csv
 test_csv:
     - !ref <data_folder>/test-clean.csv
     - !ref <data_folder>/test-other.csv
-ckpt_interval_minutes: 15 # save checkpoint every N min
 
 # Training parameters
-number_of_epochs: 50
-batch_size: 8
-lr: 1.0
+number_of_epochs: 110
+batch_size: 12 # This works for 2x GPUs with 32GB
 ctc_weight: 0.3
-gradient_accumulation: 4
+gradient_accumulation: 2
 gradient_clipping: 5.0
 loss_reduction: 'batchmean'
 sorting: random
 
+# stages related parameters
+stage_one_epochs: 90
+lr_adam: 1.0
```

Comment:
adam and 1.0 ??

Reply:
I did not change that maybe @JianyuanZhong knows more

Reply:
I think it is because this recipe uses Warmup LR schedule see NoamScheduler